src/Api/HttpApi.php

```diff
@@ -63,7 +63,7 @@ protected function hydrateResponse(ResponseInterface $response, string $class)
             return $response;
         }
 
-        if (200 !== $response->getStatusCode() && 201 !== $response->getStatusCode()) {
```

Comment:
When we import a file to the API, it returns 202 code in case of successful response, so I was needed to adjust this a little bit

Reply:
![Screenshot from 2020-10-16 22-30-02](https://user-images.githubusercontent.com/315338/96301013-34180a80-0fff-11eb-935b-e5ebe5731974.png)